bin/presubmit.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:async';
+import 'utils.dart';
+
+Future<Null> main() async {
+  final PresubmitUtils utils = new PresubmitUtils();
+
+  await utils.fetchConfiglet();
+
+  print('Formatting config.json...');
+  await utils.run('bin/configlet', ['fmt', '.']);
```

Comment:
How do we know that the `fetchConfiglet` was a success? Shouldn't there be a check before executing `configlet fmt .` ?